shopfloor/services/cluster_picking.py

```diff
@@ -776,8 +776,11 @@ def scan_destination_pack(self, picking_batch_id, move_line_id, barcode, quantit
                 qty_done=quantity,
             )
         move_line.write({"qty_done": quantity, "result_package_id": bin_package.id})
-
-        zero_check = move_line.picking_id.picking_type_id.shopfloor_zero_check
+        # Only apply zero check if the product is not a consumable.
+        zero_check = (
+            move_line.product_id.type != "consu"
```

Comment:
i'd say we only apply this check if product type is `product`
There's also service products.
Odoo or another module might add more.